scripts/migrations/30-add-account-versioning.py

```diff
@@ -0,0 +1,23 @@
+"""
```

Comment:
@chefsale @mhalambek do you want this script to be part of nearcore or part of near-ops? Also @EgorKulikov the naming of this file is a bit confusing -- the number is used to mean the new protocol version. We can either drop it or change it to 107

Reply:
Name changed

Reply:
Not really sure what is the use case  of this script. I see its a one time migration, but not sure what else would we use it in near-ops. If its just a one off migration it can stay in nearcore